samples/system-test/inspect.test.js

```diff
@@ -72,15 +73,20 @@ test(`should report string inspection handling errors`, async t => {
   t.regex(output, /Error in inspectString/);
 });
 
-// inspect_file
-test(`should inspect a local text file`, async t => {
-  const output = await tools.runAsync(`${cmd} file resources/test.txt`, cwd);
+test(`dlp_inspect_text_file`, async t => {
+  const output = await tools.runAsync(
+    `node dlp_inspect_text_file.js ${PROJECT_ID} resources/test.txt`,
+    cwd
+  );
   t.regex(output, /Info type: PHONE_NUMBER/);
   t.regex(output, /Info type: EMAIL_ADDRESS/);
 });
 
-test(`should inspect a local image file`, async t => {
-  const output = await tools.runAsync(`${cmd} file resources/test.png`, cwd);
+test(`dlp_inspect_image_file`, async t => {
+  const output = await tools.runAsync(
```

Comment:
We should use common tools instead of helper routines from `repo-tools`.

Reply:
out of scope of this PR

Reply:
Done